### ISSUE TITLE
Duplicate of #78 Question about DM0 training loss (L_AR vs L_FM in code)

### ISSUE THREAD

**Issue by ZWenyue** [dexmal/dexbotic#79]
Hi, thanks a lot for open-sourcing this awesome project!
I’ve been going through the code and the paper, and had a small question about the training loss.
From the paper, I understand that the total loss is defined as：
L_total​=λ*L_AR​+L_FM

But when I looked into the forward implementation, it seems like only the flow matching part is used:
```
v_t = self.model.action_out_proj(suffix_out_final)
action_loss = F.mse_loss(v_t, u_t, reduction="mean")
loss = action_loss
```
I didn’t notice any cross-entropy / AR loss being computed (and labels also doesn’t seem to be used here).

Is the AR loss implemented somewhere else?
Or is this corresponding to a later training stage where only flow matching is used?
